Dockerfile.s390x

```diff
@@ -132,7 +132,7 @@ RUN set -x \
 # Install notary and notary-server
 ENV NOTARY_VERSION v0.3.0
 RUN set -x \
-	&& export GO15VENDOREXPERIMENT=1 \
+	&& export GO15VENDOREXPERIMENT=0 \
```

Comment:
Yes, this makes sense. Therefore I added:
```
# Install notary and notary-server
#
# Note: We have to explicitly set GO15VENDOREXPERIMENT=0 because gccgo does not
# support vendoring: https://github.com/golang/go/issues/15628
```